tests/integration/test_project.py

```diff
@@ -14,6 +15,14 @@
 FIXTURE_ROOT = os.path.join(os.path.dirname(__file__), 'fixtures')
 
 
+@contextlib.contextmanager
```

Comment:
@elventear you can use the `containername` helper to construct your test container names (see: https://github.com/lxdock/lxdock/blob/bd9d01152e08dc828d8e87de8ef267e35106fd94/lxdock/test/testcases.py#L40).